### PR TITLE
added firebase analytics

### PR DESCRIPTION
**Pull request by sgehrman**
I didn't test this since I don't have the firebase access.

In app/build.gradle I added this:
implementation 'com.google.firebase:firebase-analytics:17.2.3

Got that online somewhere, let me know if this is unecessary.

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -61,4 +61,5 @@ flutter {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation 'com.google.firebase:firebase-analytics:17.2.3'
```
Comment (collinjackson) [inline android/app/build.gradle]: I am surprised you need this.... it should be added by the plugin automatically. You might try removing it. But it's fine by me if it's required for some reason.

**Review by collinjackson — APPROVED** [review]

**Comment by collinjackson** [comment]
I'll add you to the Firebase console.

File: android/app/build.gradle (hunk 1 above)
Comment (sgehrman) [reply android/app/build.gradle]: https://firebase.google.com/docs/android/setup

Step 4.

Only reason I ask is I have no idea how gradle works.

Comment (collinjackson) [reply android/app/build.gradle]: I think you should be able to remove it, those instructions don't apply to Flutter apps.

**Comment by collinjackson** [comment]
> I'll add you to the Firebase console.

pinged James about this

**Pull request merged**